design-docs/improved-console.md

```diff
@@ -0,0 +1,283 @@
+# Improved Console Output
+
+## Primary Goals
+* Improve perceived performance by making user aware of background work
+* Modern feel
+* User extensibility
+
+### User Goals for Console
+In priority order, the user wants to answer the following questions:
```

Comment:
Mostly that those 2 priorities are actually high on users' lists and we're addressing them is more important than the ordering. How would you change the look of the Console if these goals were reordered?